src/js/jsx/search/SearchBar.jsx

```diff
@@ -407,20 +420,24 @@ define(function (require, exports, module) {
          * Perform action based on ID
          *
          * @private
-         * @type {Datalist~onInputChange}
```

Comment:
I think @shaoshing wants that annotation there to indicate that this function is used as the `onInputChange` handler to the child `Datalist` instance.